project/urls.py

```diff
@@ -31,7 +31,7 @@
 urlpatterns = [
     url(r"^favicon\.ico$", favicon_view),
     url(r"^admin/", admin.site.urls),
-    url(r"^api/", include(api_urls)),
+    url(r"^api/", include((api_urls, 'api'))),
```

Comment:
Are you not able to define `app_name="api"` in the URLConf like you were for `web` and `exp`?

Reply:
correct, this had to do with the way the urlpatterns were generated differently in the api vs. web/exp.